yelp/obj/rating.py

```diff
@@ -8,10 +8,15 @@ class Rating(ResponseObject):
         'rating'
     ]
 
+    # TODO: add option for small/large photos
     def __init__(self, response):
         super(Rating, self).__init__(response)
 
-        # TODO: add option for small/large photos
-        self.img_url = (
-            response['rating_image_url']
-            if 'rating_image_url' in response else None)
+        # Review response uses 'rating_image_url' while Business response uses
```

Comment:
Thanks for adding the comment to explain. This is a good case of where this exception makes no sense just by reading the code & requires a comment.